docs/contributing/setup.md

```diff
@@ -135,6 +135,22 @@ Python 2.7.13
 There are also [additional resources](tooling.md) to configure your favorite
 editor to work nicely with the GitHub Desktop repository.
 
+## Node Version
+In order to build the application your node version should be `>=8.10`.
+
+First install [Homebrew](https://brew.sh/) using the following script:
```

Comment:
These installation instructions do not work for everyone. I would prefer if we defer to node's installation instructions here.